packages/content/src/attributes/types/birth/BirthDate.ts

```diff
@@ -37,6 +38,29 @@ export class BirthDate extends AbstractComplexValue implements IBirthDate {
     @validate()
     public year: BirthYear;
 
+    protected static override postFrom<T extends Serializable>(value: T): T {
+        if (!(value instanceof BirthDate)) {
+            throw new ValidationError(
+                BirthDate.name,
+                `${nameof<BirthDate>((x) => x.day)}, ${BirthDate.name}.${nameof<BirthDate>((x) => x.month)} or ${BirthDate.name}.${nameof<BirthDate>((x) => x.year)}`,
+                "An unexpected error has occured with a BirthDate."
+            );
+        }
+
+        const dateTime = DateTime.fromObject({ day: value.day.value, month: value.month.value, year: value.year.value });
+        const isValid = dateTime.isValid;
+
+        if (!isValid) {
+            throw new ValidationError(
+                BirthDate.name,
+                `${nameof<BirthDate>((x) => x.day)}, ${BirthDate.name}.${nameof<BirthDate>((x) => x.month)} or ${BirthDate.name}.${nameof<BirthDate>((x) => x.year)}`,
```

Comment:
I think we should only use `BirthDate.name` here as we cannot specify which caused the error

Reply:
The error message then looks like this: "BirthDate. :: ..."

I don't like it when there is a dot after the BirthDate without a property following it. Is there a way to remove the dot?

Reply:
argh, that's ugly..

Reply:
wtb this:

```
export class ValidationErrorWithoutProperty extends ValidationError {
    public constructor(type: string, reason: string, cause?: Error) {
        super(type, "", reason, cause);

        this.message = `${type} :: ${reason}`;
    }
}
```

if you use this you can yeet the property completely

Reply:
Sounds good, in which file/which location would you implement that class?

Reply:
`src/ValidationErrorWithoutProperty.ts`